database_wrapper.py

```diff
@@ -322,12 +329,46 @@ def create_AudioSampleMetaData_table(self) -> None:
 
         self.AudioSampleMetaData.__table__.create(bind=self.engine)
 
+    def save_question_answer_pair(self, qa_dict: dict) -> bool:
+        """
+        Save the given question answer pair into the database.
+
+        Example input:
+        {
+            "can_we_answer": False,
+            "answer_type": AnswerType.other,
+            "question_format": "What is the meaning of life?",
+            "answer_format": "Dr. Fizzbuzz says the answer is sqrt(1764)"
+        }
+
+        Args:
+            qa_dict: a dictionary that corresponds to the fields in QuestionAnswerPair  # noqa
+
+        Raises:
+            BadDictionaryKeyError - ...
+            BadDictionaryValueError - ...
+
+        Returns:
+            True if all is good, else False
+        """
```

Comment:
I love the long description of inputs and outputs! Should be very helpful for future maintenance.

Reply:
<3